packages/core/sdk/src/auth/internal/CoreStitchAuth.ts

```diff
@@ -458,20 +462,10 @@ export default abstract class CoreStitchAuth<TStitchUser extends CoreStitchUser>
    * Processes the response of the login/link request, setting the authentication state if appropriate, and
```

Comment:
[nit] move this comment to `processLoginResponse`, and if you have time add a comment here describing what this function does now